### DISCUSSION TITLE
CV: Multi-threading error

### DISCUSSION THREAD

**Discussion by masadshoaib** [Nixtla/statsforecast#883] [Q&A] [answered]
Hi,

I am running the built-in command of cross validation in statsforecast and am getting the error below. has anyone faced this one before?

/usr/lib/python3.10/multiprocessing/popen_fork.py:66: RuntimeWarning: os.fork() was called. os.fork() is incompatible with multithreaded code, and JAX is multithreaded, so this will likely lead to a deadlock.
  self.pid = os.fork()

I am running the code below for this:
sf = StatsForecast(
        models=[AutoARIMA(season_length = season_length),
      AutoETS(),
      AutoCES(),
      AutoTheta(),],
        freq='MS',
        fallback_model=SeasonalNaive(season_length=season_length),
        n_jobs=-1
    )
    crossvalidation_df = sf.cross_validation(
        df=train_data,
        h=12,
        step_size=12
    )

**Comment by jmoralez** [comment, marked as answer]
Hey. That's not an error, that's a warning. Is that the full code you're using? We don't depend on JAX, so it's very strange that you get that warning, unless you're also importing it.

**Reply by masadshoaib** [reply]
Hmm you are correct. I am not importing it but have plenty of other models imported and being run from different packages. Maybe that dependency is being installed as part of some other package. This is working alright if I run in a separate notebook.

Thank you for your quick support @jmoralez. Highly appreciated!
